packages/theme/src/theme/_navbar.scss

```diff
@@ -25,7 +25,6 @@
 		display: flex;
 
 		li {
-			display: flex;
 			align-items: center;
```

Comment:
I don't think this does anything without flex? Can we not remove the whole `li` block?

Reply:
You're probably right, although I wanted to make as less moves  as possible on this since I can't test storybook, updating PR